docs/hazmat/primitives/asymmetric/ed25519.rst

```diff
@@ -103,6 +103,12 @@ Key interfaces
 
         :return bytes: Serialized key.
 
+.. class:: Ed25519PrivateKeyWithSerialization
```

Comment:
We don't need these (they're legacy we've deprecated on our other classes)

Reply:
that leaves a question of how you want the serialization docs to look.  would you rather i change all the allowed types to remove the "withserialization" aliases or just leave it so that ed448/ed25519 private keys are referenced directly and Ec/RSA/DSA keys are still referred to by their alias? i just think it would look weird for some private key types to be "with serialization" and some not.

Reply:
ok this is the way that makes the most sense to me, given that these are the types that are actually checked for, but it does rather broaden the scope of the pr to change them here. please advise at your convenience.